src/ontology/mondo-edit.obo

```diff
@@ -284128,15 +284128,15 @@ property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/4
 id: MONDO:0014200
 name: aldosterone-producing adenoma with seizures and neurological abnormalities
 subset: ordo_disease {source="Orphanet:369929"}
-synonym: "aldosterone-secreting adenoma" EXACT [Orphanet:85142]
+synonym: "aldosterone-secreting adenoma" RELATED [Orphanet:85142]
```

Comment:
@sabrinatoro I am going to run some questions by you, to make sure you are in agreement:

1. This term: MONDO:0014200 aldosterone-producing adenoma with seizures and neurological abnormalities- there is an xref to Orphanet:85142 {source="MONDO:relatedTo"} but this term no longer exists in Orphanet. I am not sure if we have an SOP for dealing with these. I don't think it is MONDO:equivalentObsolete (live class in Mondo and an obsolete class in the source ontology) because I don't see it as being obsolete in Orphanet. I have no idea where this came from. Do we just want to leave it as MONDO:relatedTo? Or remove it?

I think that since this is not an equivalent xref to Mondo, the syonyms should not be equivalent, so I changed them to RELATED synonyms here. Do you agree with this?